test/component_test.go

```diff
@@ -0,0 +1,51 @@
+package test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/cloudposse/test-helpers/pkg/atmos"
+	helper "github.com/cloudposse/test-helpers/pkg/atmos/component-helper"
+	awshelper "github.com/cloudposse/test-helpers/pkg/aws"
+	"github.com/stretchr/testify/assert"
+)
+
+type ComponentSuite struct {
+	helper.TestSuite
+}
+
+func (s *ComponentSuite) TestBasic() {
+	const component = "ec2-instance/basic"
+	const stack = "default-test"
+	const awsRegion = "us-east-2"
+
+	defer s.DestroyAtmosComponent(s.T(), component, stack, nil)
+	options, _ := s.DeployAtmosComponent(s.T(), component, stack, nil)
+	assert.NotNil(s.T(), options)
+
+	instanceIds := atmos.OutputList(s.T(), options, "instance_id")
```

Comment:
Is there any reason we should use OutputList instead of Output ?
```suggestion
	instanceIds := atmos.Output(s.T(), options, "instance_id")
```

Reply:
My rationale was that `module.ec2_instance[*].id` returns a list of instance IDs, and the `Output()` function only supports strings, numbers, or bools. Since we’re outputting a list, I assumed `OutputList()` would work as intended instead.